.gitignore

```diff
@@ -2,4 +2,5 @@ env
 venv
 skills_DS/venv
 __pycache__/
-docker/postgresql/data
\ No newline at end of file
+secrets.json
```

Comment:
Nice

Reply:
We should have a `secrets_example.json` file which dummy data for reference sake.